portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,17 @@
+package com.google.sps.data;
```

Comment:
You can look at project lombok.

You dont need to create any new constructors for your class then. It seems overkill here. But if you have more fields, writing constructors can be a pain.

Reply:
Lombok allows, you to just write annotations on top and code gets auto generated